solutions/cf-1713E.mdx

```diff
@@ -0,0 +1,164 @@
+---
+id: cf-1713E
+source: Codeforces Div 2
+title: Cross Swapping
+author: Aakash Gokhale
+---
+
+## Explanation
+
+[Official Analysis](https://codeforces.com/blog/entry/105419)
+
+Notice that $a[i][j]$ can only be swapped with $a[j][i]$ no matter which operations are performed.
+
+In addition, we want to lexicographically sort the array. Thus, we can iterate and greedily process the array.
```

Comment:
> Thus, we can iterate and greedily process the array.

what does this mean

Reply:
Since we want the lexicographically smallest answer, the earlier indices matter more than later indices so we find optimal row swaps for earlier indices first.

Reply:
then say that

Reply:
fixed